database/index.js

```diff
@@ -0,0 +1,44 @@
+//in here set up mongoose database
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+// var bodyParse = require('body-parser');
+mongoose.connect('mongodb://localhost/helmets', { useNewUrlParser: true });
+
+//this is the piece I was missing to be able to use the mongoose connect methods.
+var db = mongoose.connection;
+
+var dropDB = function() {
+  //access the DB as an array
+  db.collections['helmets'].drop(function(err) {
+    console.log('HELMETS DROPPED');
+  });
+};
+
+var helmetsSchema = new Schema({
+  id: Number,
+  productName: String,
+  colors: [],
```

Comment:
It'd be more descriptive to include the data type for each color in the colors array. For instance, with your colors being stored as hexadecimal color codes, this line could be the following to more explicitly describe what type of data is stored.

```suggestion
  colors: [String],
```
As is, it is specifying a mixed-type mongoose array.